src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -23,10 +23,9 @@
 #include "util.hpp"
 
 // required for get_properties_per_device()
-#include <intel_npu/al/config/config.hpp>
-#include <intel_npu/al/config/npuw.hpp>
-#include <npuw_private_properties.hpp>
-
+#include "config/config.hpp"
```

Comment:
Can you consider putting the headers of the new `common` library into nested folder like this `intel_npu/common`. It will make it clear that the headers are the part of a separate library.

I think it is a common practice for our code base. Examples:
* https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_npu/src/utils/include/intel_npu/utils
* https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_npu/src/al/include/intel_npu/al

Reply:
done